src/parser/jobs/gnb/modules/Defensives.ts

```diff
@@ -3,7 +3,7 @@ import {Defensives as CoreDefensives} from 'parser/core/modules/Defensives'
 export class Defensives extends CoreDefensives {
 	protected override trackedDefensives = [
 		this.data.actions.SUPERBOLIDE,
-		this.data.actions.NEBULA,
+		this.data.actions.GREAT_NEBULA,
```

Comment:
Not something we need to address here, but it's probably worth sticking a pin in this one to revisit if and when we implement the level sync logic we were discussing on Discord.